include/pingcap/kv/Cluster.h

```diff
@@ -36,7 +38,14 @@ struct Cluster
         , rpc_client(std::make_unique<RpcClient>(config))
         , oracle(std::make_unique<pd::Oracle>(pd_client, std::chrono::milliseconds(oracle_update_interval)))
         , lock_resolver(std::make_unique<LockResolver>(this))
-    {}
+    {
+        if (config.api_version == "v1")
```

Comment:
Did we convert config to lower case before compare?

Reply:
Yes, the caller must be set the api version config in lowwer case.